bitmovin/resources/models/encodings/muxings/__init__.py

```diff
@@ -8,3 +8,4 @@
 from .progressive_mov_muxing import ProgressiveMOVMuxing
 from .information import ByteRange, MuxingInformationAudioTrack, MuxingInformationVideoTrack, \
     ProgressiveTSInformation, MP4MuxingInformation
+from .broadcast_ts import *
```

Comment:
why you are using a wildcard import here? the other imports in this file look completely different?